lua/plugins/autocomplete/lspconfig/main.lua

```diff
@@ -13,9 +13,12 @@ local on_attach = function(client, bufnr)
     vim.api.nvim_create_autocmd({ "CursorHold", "CursorHoldI" }, {
       buffer = bufnr,
       group = "LspDocumentHighlight",
-      callback = function()
-        vim.lsp.buf.document_highlight()
-      end,
+      callback = vim.lsp.buf.document_highlight,
+    })
+    vim.api.nvim_create_autocmd("CursorMoved", {
```

Comment:
If the on_attach function enables document highlights, it should also clean them up on detachment to avoid lingering highlights.